Monika After Story/game/script-compliments.rpy

```diff
@@ -888,6 +889,174 @@ label mas_compliment_humor_3:
+            _("We're so lucky to have one another!"),
+            _("We don't need to feel lonely anymore!"),
+            _("Let's enjoy our time together today!"),
+            _("I really appreciate you, [player]!"),
+            _("Thanks for making time for me!"),
+            _("I couldn't wait for you to get back!"),
+            _("I was lonely waiting for you!"),
+            _("I've been thinking of you while you were gone!"),
+        ]
+        missed_quip = random.choice(missed_quips)
+    $ store.mas_lockEventLabel("mas_compliment_missed", eventdb=store.mas_compliments.compliment_database)
+    if mas_isMoniNormal(higher=True):
+        $ mas_gainAffection(2)
+        $ hugchance = 2
+
+        #Really short absences
+        if mas_getAbsenceLength() < datetime.timedelta(hours=1):
+            $ hugchance = 0
+
+        if mas_getAbsenceLength() >= datetime.timedelta(days=3):
+            $ missedchance = renpy.random.randint(1,3)
+            if missedchance == 1:
+                m 6wub "I missed you a ton, [player]!"
+            elif missedchance == 2:
+                m 6wub "I missed you a whole lot, [player]!"
+            else:
+                m 6wubfb "...I ree{w=0.1}ee{w=0.1}ee{w=0.1}ally missed you [player]!"
+            $ hugchance = 17
+            if mas_getAbsenceLength() >= datetime.timedelta(days=7):
+                $ hugchance = 50
+        else:
+            m 6eub "I missed you too, [player]!"
+
+        show monika 2hublb
+        $ renpy.say(m, missed_quip)
+        if mas_isMoniEnamored(higher=True) and renpy.random.randint(1,50) <= hugchance:
+            m 5rsa "Say, [player]."
+            m 5rsc "I was kinda hoping that..."
+            m 5fsc "You know, since it's been a little while..."
+            m 2fkblc "Could you give me a hug? I've been feeling pretty lonely while you were gone.{nw}"
+            $ _history_list.pop()
+            menu:
+                m "Could you give me a hug? I've been feeling pretty lonely while you were gone.{fast}"
+                "Okay.":
+                    call monika_holdme_prep(lullaby=MAS_HOLDME_NO_LULLABY, stop_music=True, disable_music_menu=True)
+                    call monika_holdme_start
+                    call monika_holdme_end
+
+                    m 2fkblb "Mmm... that was really nice, [player]."
+                    m 2ekbsa "You really know how to make a girl feel special!"
+                    m 7rublsdra "Anyway..."
+                    m 1esa "What shall we do today, [mas_get_player_nickname()]?"
+
+                    $ mas_gainAffection(0.75)#hug aff
+                    return
+                "Not right now.":
+                    #a lil sad
+                    $ mas_loseAffection()
+                    m 6ekc "...alright, [player]. Maybe later, then?"
+    #Base negative responses on monika_love label
+    elif mas_isMoniUpset():
+        $ mas_gainAffection()
+        python:
+            missed_quips_upset_cynical = [
```

Comment:
There's an unspoken agreement that constants are being defined at the top. Let's move all these quips at the top of the label, so it's easier to edit/review them later.

Reply:
Change made in https://github.com/Monika-After-Story/MonikaModDev/pull/7419/commits/a70b95c814f202d80af1f694e15d841725713e31.